hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/NonblockingConsumersSupervisor.java

```diff
@@ -86,9 +86,10 @@ public void assignConsumerForSubscription(Subscription subscription) {
         logger.info("Creating consumer for {}", subscription.getQualifiedName());
         try {
             Consumer consumer = consumerFactory.createConsumer(subscription);
-            logger.info("Created consumer for {}", subscription.getQualifiedName());
+            Signal start = Signal.of(Signal.SignalType.START, subscription.getQualifiedName(), consumer);
+            logger.info("Created consumer for {}. Initialized start signal with id {}", subscription.getQualifiedName(), start.getId());
```

Comment:
Ok, after your suggestion I have changed all logs containing signals to format like this:
```
2017-06-20 14:52:24.555 INFO  p.a.t.h.c.s.process.ConsumerProcess - Starting consumer for subscription group.topic$sub. [Signal(1, START)]
2017-06-20 14:52:24.562 INFO  p.a.t.h.c.s.process.ConsumerProcess - Started consumer for subscription group.topic$sub in 0ms. [Signal(1, START)]
2017-06-20 14:52:24.562 INFO  p.a.t.h.c.s.process.ConsumerProcess - Stopping main loop for consumer group.topic$sub. [Signal(2, STOP)]
```